R-package/src/Makevars

```diff
@@ -0,0 +1 @@
+
```

Comment:
these address warnings from CRAN about having `Make` scripts with only LF characters in them